src/Internal/Serialization/ToBytes.purs

```diff
@@ -0,0 +1,46 @@
+module Ctl.Internal.Serialization.ToBytes (toBytes) where
+
+import Ctl.Internal.Serialization.Address (Address)
+import Ctl.Internal.Serialization.Types
+  ( AuxiliaryDataHash
+  , DataHash
+  , Ed25519Signature
+  , GenesisDelegateHash
+  , GenesisHash
+  , NativeScript
+  , PlutusData
+  , Redeemers
+  , ScriptDataHash
+  , Transaction
+  , TransactionBody
+  , TransactionHash
+  , TransactionOutput
+  , TransactionUnspentOutput
+  , TransactionWitnessSet
+  , Value
+  )
+import Ctl.Internal.Types.ByteArray (ByteArray)
+import Untagged.Union (type (|+|))
+
+-- NOTE returns cbor encoding for all but hash types, for which it returns raw bytes
```

Comment:
I think this is another reason to have a typeclass for this, maybe with a fundep for the bytes type (maybe that's overcomplicated). We already have a `FromBytes` so a `ToBytes` would be best

No need to address that in this PR though